integreat_cms/cms/migrations/0080_alter_poi_category.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.2.20 on 2023-07-25 15:46
+
+from django.db import migrations, models
+
+from ..constants import poicategory
+
+
+# pylint: disable=unused-argument
+def check_and_update_poicategory(apps, schema_editor):
+    """
+    This function checks whether the default POI category "Other" exists and creates it if not.
```

Comment:
Probably I'm missing something, but if we have one set of categories for the entire CMS (not separate sets for each region), can we just check in the prod database, if we already have this category?

Reply:
Like the language objects, even though the default category currently exists, it can have been deleted when this migration runs in future :/

Reply:
Also the migration file should work on all systems, even new production systems we might set up in the future for additional whitelabeling versions (e.g. aschaffenburg).